pullauta.default.ini

```diff
@@ -0,0 +1,231 @@
+# so these are only partly counted
+firstandlastreturnfactor=1
+
+# green weighting for last return - these may be vegetation but less likely that earlier returns
+lastreturnfactor =1
+
+firstandlastreturnasground=3
+# green values for triggering green shades. Use high number like 99 to avoid some of the shades.
+#greenshades=0.0|0.1|0.2|0.3|0.4|0.5|0.6|0.7|0.8|0.9|1.0|1.1|1.2|1.3|1.4|1.5|1.6|1.7|1.8|1.9|2.0|2.1|2.2|2.3|2.4|2.5|2.6|2.7|2.8|2.9|3.0
+
+greenshades=0.2|0.35|0.5|0.7|1.3|2.6|4|99|99|99|99
+
+# tone for the lightest green. 255 is white.
+lightgreentone=200
+
+# dont change this now
+greendotsize=0
+
+# block size for calculating hits-below-green ratio. use 3 if  greendetectsize is smaller than 5, if
+# it is bigger then use 1
+groundboxsize=1
+
+# green raster image filtering with median filter. Two rounds
+# use 1 to do no filtering.
+medianboxsize=9
+medianboxsize2=1
+
+## yellow parameters
+### hits below this will be calculated as yellow
+yellowheight=0.9
+
+### how big part or the points must be below yellowheight to trigger yellow
+yellowthresold=0.9
+
+#############################################
+## cliff maker min height values for each cliff type. vertical drop per 1 meter horisontal distance
+##  cliff1 = these cliffs will be erased if steepness is bigger than steepness value below
+##  cliff2 = impassable cliff
+
+cliff1 = 1.15
+cliff2 = 2.0
+cliffthin=1
+
+cliffsteepfactor=0.38
+cliffflatplace=3.5
+cliffnosmallciffs=5.5
+
+cliffdebug=0
+## north lines rotation angle (clockwise) and width. Width 0 means no northlines.
+northlinesangle=0
+northlineswidth=0
+
+## Form line mode, options:
+# 0 = 2.5m interval, no formlines
+# 1 = 2.5m interval, every second contour thin/thick
+# 2 = 5m interval, with some dashed form lines in between if needed
+
+formline=2
+
+# steepness parameter for form lines. Greater value gives more and smaller value gives less form lines.
+formlinesteepness=0.37
+
+## additional lengt of form lines in vertex points
+formlineaddition=17
+
+## shortest gap in between form line ends in vertex points
+minimumgap = 30
+
+# dash and gap parameters for form lines
+dashlength = 60
+gaplength =12
+
+# interval for index contours. Used only if form line mode is 0
+indexcontours=12.5
+
+# smoothing contrors. Bigger value smoothes contours more. Default =1. Try values about between 0.5 and 3.0
+smoothing = 0.7
+
+# curviness. How curvy contours show up. default=1. Bigger value makes more curvy/exaggerated curves (reentrants and spurs)
+curviness=1.1
+
+# knoll qualification. default =0.8. range 0.0 ... 1.0  Bigger values gives less but more distinct knolls.
+knolls=0.6
+
+# xyz factors, for feet to meter conversion etc
+coordxfactor=1
+coordyfactor=1
+coordzfactor=1
+
+# las/laz to xyz thinning factor. For example 0.25 leaves 25% of points
+thinfactor = 1
+
+# if water classified points, this class will be drawn with blue (uncomment to enable this)
+# waterclass=9
```

Comment:
> I am thinking about possible break in behaviour...

I guess it would be kind of breaking since the config file does not specify any value and thus those two config values would have their defaults, which would then change if we change it... Especially since we are using it as an "on/off" flag and checking `if water > 0` in `vegetation.rs` line 497, it would then be on by default. Maybe we need two different config values with different meaning? Like one that defines which class is water (default to 9) and one which turns on this detection in `vegetation.rs`. It would still be somewhat of a breaking change though... 🤔